js/foundation.util.touch.js

```diff
@@ -3,9 +3,9 @@
 //**Done by Yohai Ararat ***************************
 //**************************************************
 
-class Touch {}
+import $ from 'jquery';
 
-(function($) {
+class Touch {}
```

Comment:
Should this really be closed immediately? The part there after is indented and looks like it should be part of this class.